sdk-setup/src/rpmvalidation

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+#
+# Copyright (C) 2018 Jolla Ltd.
+# Contact: Ville Nummela <ville.nummela@jolla.com>
+#
+# This file is part of the SailfishOS SDK
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+usage() {
+    cat <<EOF
+Run a basic quality criteria check for the given RPM file.
+
+Usage: 
+    $0 -t <target> [OPTIONS] <rpm-file>
+    
+Options:
+    -t|--target <target> The OS version to validate for. 
+                         Accepts the same argument as 'sb2 -t' does.
+    -l|--list-suites     Lists available suites
+    --suites <suite1,suite2,...> Use specified suites.
+                         If omitted, all suites are used.
+EOF
+}
+
+CONFIG_DIRS=({/usr/share,/etc}/rpmvalidation/suites)
+
+SUITES=()
+
+US=$'\x1F'
+write() ( IFS=$US; printf '%s\n' "$*"; )
+
+# Write an error message in a format that Qt Creator understands.
+# See merrpmvalidationparser.cpp for details
+error() {
+    local section=$1
+    local filename=$2
+    shift 2
+    echo "$section"
+    echo "${section//?/=}"
+    echo "ERROR [$filename] $*"
+    echo "FAILED"
+    exit 1
+}
+
+lacks_element() {
+    local match=$1
+    shift
+    for element in "$@"; do
+        if [[ $match == "$element" ]]; then
+            return 1
+        fi
+    done
+    return 0
+}
+
+populate_suites() {
+    local target=$1
+    local validator_inis=()
+    local uniquie_inis=()
+    SUITES=()
+    validator_inis=($(sb2 -t "$target" find "${CONFIG_DIRS[@]}" -type f -printf %f\\n 2>/dev/null))
```

Comment:
Yup, good idea. It would break on ini file under subdir though, so we should either add `-maxdepth 1` or use `%P` instead of `%f`. We could also add `-name "*.ini"`. Feel free to use quotes instead of escaping :)